SPECS/rpc/rpc.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
   "Signatures": {
-    "rpc-go-2.45.1.tar.gz": "54dcb9a72593f7eef02a35255bc5f79636fd7832d26b0fb1189ad300259178f9"
+    "rpc-go-2.48.9.tar.gz": "e8162c735a87b1ffebc18a2c2a5c82d0de5f03311573b266e093162b58f7fd43"
```

Comment:
the rpc-go-2.48.9.tar.gz checksum is failing to hydrate in build #1343,
download the source tarball and check the integrity of the content